### PR TITLE
Squashed commit of the following:

### PR DESCRIPTION
**Pull request by MrPewPewLaser**
commit 2f3129b5610f6515ad36a1c3a13f600049702556
Author: unknown <veit.caspers@web.de>
Date:   Tue Nov 8 21:48:59 2022 +0100

    factions defines arsenal fixes reward

### CODE REVIEW

File: Functions/AI/fn_main_ao_garrison_spawner.sqf [hunk 1]
```diff
@@ -76,10 +76,18 @@ private _buildingBlacklist = [
     "Land_Lighthouse_small_ruins_F"
 ];
 
+
```
Comment (FlyingThread) [inline Functions/AI/fn_main_ao_garrison_spawner.sqf:79]: Why the three blank lines?
Not really a blocker, just curious

File: Functions/AI/fn_main_ao_garrison_spawner.sqf [hunk 2]
```diff
@@ -171,7 +179,7 @@ for "_i" from 1 to _buildingCount do {
     InA_Server_mainAOUnits append _x;
 } forEach [_units, _fortObjs];
 
-// Set garrisoned building to invulnerable for the duration of the AO
+/*Set garrisoned building to invulnerable for the duration of the AO
```
Comment (FlyingThread) [inline Functions/AI/fn_main_ao_garrison_spawner.sqf:182]: Do you want to enable building destruction during an AO?

**Review by FlyingThread — COMMENTED** [review]
Some blank line changes that could use clean-up.
Only one open question.

File: Functions/AI/fn_main_ao_garrison_spawner.sqf (hunk 1 above)
Comment (MrPewPewLaser) [reply Functions/AI/fn_main_ao_garrison_spawner.sqf:79]: idk, probably just left over stuff

File: Functions/AI/fn_main_ao_garrison_spawner.sqf (hunk 2 above)
Comment (MrPewPewLaser) [reply Functions/AI/fn_main_ao_garrison_spawner.sqf:182]: Yes, that is the plan

**Review by FlyingThread — APPROVED** [review]

**Pull request merged**